src/org/jitsi/impl/neomedia/RTPConnectorOutputStream.java

```diff
@@ -123,6 +140,12 @@
 
         PACKET_QUEUE_CAPACITY
             = packetQueueCapacity >= 0 ? packetQueueCapacity : 256;
+
+        logger.debug("Initialized configuration. "
```

Comment:
We usually do the logging of constructed strings of logger.isDebugEnabled(). I suppose it doesn't matter too much anyway.